Rakefile

```diff
@@ -59,8 +59,12 @@ namespace :arm do
         md = REGEN_METADATA[dir.to_sym]
         ar_base_command = "#{OS.windows? ? '' : 'mono '} #{REGEN_METADATA[:autorest_loc]}"
 
-        command = "#{ar_base_command} -i #{md[:spec_uri]} -pv #{md[:version]} -n #{md[:ns]} -pn #{md[:pn].nil? ? dir : md[:pn]} -g Azure.Ruby -o lib"
-        command += " -m #{md[:modeler]}" unless md[:modeler].nil?
+        if md[:spec_uri].end_with?('.json')
```

Comment:
this is fine for now, though I heard Garrett said that they are removing the legacy command line soon, when that happens, this won't work, please check with him when that will occur, so this can be updated.